### ISSUE TITLE
Team was created but is now not visible so can't be managed

### ISSUE THREAD

**Issue by onXpdavis** [mapbox/osmcha-frontend#653]
## *Brief Description*

I made a mapping team to easily manage new users and removing old users but the team is no longer visible to me so can't be managed.
## What  is the current behaviour, (attach relevant screenshots) ?

When navigating to `My Teams`, the team I created (`onXmaps Geospatial`) is not visible. 
The team can be confirmed since we can filter changesets by the team name and get the expected results.
## What  is the expected behaviour ?

Team is visible and able to be managed.
## When does this occur ?

It has occurred every time I've tried to view the team since I initially created the team. 
## How do we replicate the issue ?

1. Create a team of users, then save it.
2. Navigate to `My teams` and it won't be available; only the option to create a team is available.
3. Confirm the team exists by applying a filter for the `onXmaps Geospatial` team.
4.

## Please tell us about your environment:

Chrome Version 112.0.5615.49 (Official Build) (x86_64)

**Comment by willemarcel** [comment]
@onXpdavis are you trying to edit the team with the user `pbdavis`? Could you try with this URL https://osmcha.org/teams/282?

**Comment by onXpdavis** [comment]
> @onXpdavis are you trying to edit the team with the user `pbdavis`? 
Now that I think about it, I _think_ I did create the team with that username, but then perhaps shortly after that I changed to my current username to align with our company's standards, which is `onxmaps_pbdavis`. Perhaps that is the root of my problem?

>Could you try with this URL https://osmcha.org/teams/282?

When I follow that link I see:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/51759466/231883447-e6c86d53-1bb8-4377-ae8b-6d830d14824f.png">
Is that what you're intending for me to see? I can't make any changes to that team (`Vi`) since I didn't create it.

**Comment by onXpdavis** [comment]
> Sorry, I sent you the wrong link. It should be https://osmcha.org/teams/414

Yes that looks better! With this link I can see the team I created and it's associated with my _new_ username.

> I think there is some cache or an error in your username update. Could you try to logout and login again?

But upon logging out then back in, I am back to not being able to see my team 😢

**Comment by tordans** [comment]
I think think the bug is, that the teams dropdown only ever looks at the first page of teams, but there are in fact 9 pages now.
https://github.com/OSMCha/osmcha-frontend/blob/fbed17576726de5e193adffbe248ebd623f08af1/src/config/filters.json#L231 suggests that this odes not apply a search on the server but returns the full list of page 1 and filters that list.

A workaround is…
- select the first team
- apply the filters
- copy the URL
- use the console with `decodeURIComponent('URL')`
- right click > copy string contents (for Chrome…)
- modify the value `,"mapping_teams":[{"label":"FMC Radinfra","value":"FMC Radinfra"}]}` of the URL with the string from the team that you created
- load that URL => the team is used

**Comment by tordans** [comment]
@jake-low TBH I am not sure this is worth the effort or at least I wonder if this should be something to fix now (compared to other issues). In all my time using OSMCha I never had the need to add teams. And today I only used it as a quick way around the fact that I did not get the User filter working (which I now see is https://github.com/OSMCha/osmcha-frontend/issues/814).

When I worked on https://github.com/OSMCha/osmcha-frontend/issues/741 I wanted to leave the feature to reduce complexity.

There are probably some use cases for this, but the user filter should solve most of them.
